front/lib/api/assistant/configuration.ts

```diff
@@ -373,7 +373,7 @@ async function fetchWorkspaceAgentConfigurationsWithoutActions(
             sId: agentsGetView.agentIds.filter((id) => !isGlobalAgentId(id)),
           },
           order: [["version", "DESC"]],
-          ...(agentsGetView.allVersions ? {} : { limit: 1 }),
+          ...(agentsGetView.allVersions ? {} : { status: "active" }),
```

Comment:
When is this default view used? Are we sure there is not expectation that only one would be returned?

Reply:
This is definitely not desired as we don't want to limit to 1 row if we pass multiple ids.